contrib/hdparm/template.py

```diff
@@ -0,0 +1,20 @@
+pkgname = "hdparm"
+pkgver = "9.65"
+pkgrel = 0
+build_style = "makefile"
+make_cmd = "gmake"
+make_install_args = ["-j1", "binprefix=/usr", "sbindir=/usr/bin"]
+hostmakedepends = ["gmake"]
+makedepends = ["linux-headers"]
+pkgdesc = "Get/Set ATA/SATA drive parameters under Linux"
+maintainer = "Subhaditya Nath <sn03.general@gmail.com>"
+license = "BSD-1-Clause AND GPL-2.0-only"
+url = "https://sourceforge.net/projects/hdparm"
+source = f"$(SOURCEFORGE_SITE)/{pkgname}/{pkgname}-{pkgver}.tar.gz"
+sha256 = "d14929f910d060932e717e9382425d47c2e7144235a53713d55a94f7de535a4b"
+# no test suite
+options = ["!check"]
```

Comment:
can you try building it with CFI on? and test it a bit

Reply:
Is that necessary? These are all text files....

Reply:
I don't think anything's being compiled...? So how'd CFI help?

sorry.... i am a total noob when it comes to hardening stuff like CFI, LTO, sanitization and whatever else is there...

Reply:
what do you mean? hdparm is a C program

Reply:
Ah.. sorry... had a brainfart there 😅

Reply:
It does seem to be building with CFI enabled.... but I don't have any HDDs to test it on...